docs/maui/views/popup.md

```diff
@@ -33,6 +37,19 @@ A `Popup` can be created in XAML or C#:
 </toolkit:Popup>
 ```
 
+```csharp
+public partial class SimplePopup : Popup
+{
+    public SimplePopup()
+    {
+        InitializeComponent();
+    }
+}
+```
+
+> [!IMPORTANT]
+> If the code behind file is not created along with the call to `InitializeComponent` then an exception will be thrown when trying to display your `Popup`.
```

Comment:
Nit: maybe add more detail on _which_ exception so people know this is likely what they're seeing?

Reply:
Very good point! I am not struggling to reproduce the exception to find out that detail :D